app/server/mergedServerMain.ts

```diff
@@ -8,6 +8,7 @@
 import {FlexServer, FlexServerOptions} from 'app/server/lib/FlexServer';
 import {GristLoginSystem} from 'app/server/lib/GristServer';
 import log from 'app/server/lib/log';
+import { getGlobalConfig } from "app/server/lib/globalConfig";
```

Comment:
I'm going to stop nagging about this now, but, it would be great to copy local style when you can (in this case, imports in alphabetical order, and single quotes, and no spaces near {})

Reply:
Should be sorted :+1: